go/chat/partsource.go

```diff
@@ -0,0 +1,108 @@
+package chat
+
+import (
+	"context"
+
+	"github.com/keybase/client/go/chat/globals"
+	"github.com/keybase/client/go/chat/types"
+	"github.com/keybase/client/go/chat/utils"
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/chat1"
+	"github.com/keybase/client/go/protocol/gregor1"
+)
+
+type partDiskStorage struct {
+	Uids []gregor1.UID
+	Hash string
```

Comment:
add `HashVersion int`

Reply:
What for?

Reply:
if we added a version number on the server side

Reply:
Seems like overkill, server can generate any value it wants to.